Frigate_Camera_Notifications/Beta.yaml

```
@@ -1227,7 +1269,7 @@ action:
                                 icon: "{{icon_3}}"
                                 destructive: true
 
-                  - if: "{{tts and (not tts_helper or not alert_once or (tts_helper and id not in states(tts_helper)))}}"
+                  - if: "{{tts and not telegram and (not tts_helper or not alert_once or (tts_helper and id not in states(tts_helper)))}}"
```

I also don't see the need to say not telegram here. 

```suggestion
                  - if: "{{tts and (not tts_helper or not alert_once or (tts_helper and id not in states(tts_helper)))}}"
```